workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -332,6 +335,18 @@ private void FindTranslationUnits()
             }
         }
 
+        private void FindGameObjectTranslations()
+        {
+            var monoBehaviourTranslationTypes = AppDomain.CurrentDomain.GetAssemblies().SelectMany(assembly => assembly.GetTypes())
```

Comment:
Do you mean `gameobjectTranslationTypes`?

Reply:
Yes, I forgot to rename this across the board. Thanks for pointing out. Fixed.